app/scripts/modules/openstack/pipeline/stages/bake/openstackBakeStage.js

```diff
@@ -73,10 +79,17 @@ module.exports = angular.module('spinnaker.openstack.pipeline.stage.bakeStage',
           $scope.stage.baseOs = $scope.baseOsOptions[0].id;
         }
         $scope.viewState.roscoMode = SETTINGS.feature.roscoMode;
+        $scope.showAdvancedOptions = showAdvanced();
         $scope.viewState.loading = false;
       });
     }
 
+    function showAdvanced() {
+      let stage = $scope.stage;
+      return !!(stage.templateFileName || (stage.extendedAttributes && _.size(stage.extendedAttributes) > 0) ||
```

Comment:
Are the values other than `extendedAttributes` and `templateFileName` relevant to an Openstack bake stage?